docs/try_local_system.md

```diff
@@ -131,6 +131,11 @@ for your distribution and follow along.
    cargo install diesel_cli --no-default-features --features "postgres"
    ```
 
+5. Make sure your system has openssl installed (specially in case on `Windows`)
+   ```shell
+   winget install openssl
+   ```
+
```

Comment:
Could you also mention the package name for linux and to "install it using your package manager of choice"? I think the linux package was named `libssl-dev` for ubuntu iirc.

Reply:
It'd be great if we could also have a separate section specifically for Windows setup, since Windows could bring it's own set of issues. I'm fine with the Windows setup instructions being taken up on a separate PR.